### DISCUSSION TITLE
consistent use of architecture names at qemu images

### DISCUSSION THREAD

**Discussion by ulbu261833** [Elektrobit/ebcl_template#6] [General]
For amd64 there is used the name amd64 and the architecture x86_64 while for arm64 there is used the name aarch64 and the architecture aarch64.
I would have preferred to define either name and architecture x86_64 for amd64 when aarch64 shall continue to be used for both at arm64 or to define name and architecture differently for both amd64 and arm64 (amd64: name amd64, architecture x86_64; arm64: name arm64, architecture aarch64).

**Comment by thir820** [comment]
This caused by inconsistent upstream naming. The Debian packages use "amd64", qemu is using x86_64. Similar for aarch64 and arm64.

**Reply by ulbu261833** [reply]
Shouldn't in this case the upstream naming be made consistent? At a user guide the usage of the current naming is hard to explain...

**Reply by thir820** [reply]
It are different projects and communities, but I will decide for one consistent naming for the next SDK release.
